tests/baselines/reference/intraExpressionInferences.errors.txt

```diff
@@ -97,6 +97,14 @@ tests/cases/conformance/types/typeRelationships/typeInference/intraExpressionInf
         }
     });
     
+    test({

```

Comment:
#45255 has 2 failing test cases, the 3rd call is basically the same as the call above this one here, but the 1st call (the one that is being added here) is slightly different - it "transfers" the return type from `a` to the `c` parameter type through the identity `b`. So it computes `R1`and `R2` to be the same.

Perhaps it's not that different from the other test case but maybe it's worth adding - i will let you be the judge of that 😉